### PR TITLE
Ilianna pirounaki

### LINKED ISSUE #213: Interactive chart not available
* **Is your feature request related to a problem? Please describe.**

The interactive version of bubble charts is not working properly and we need to find a workaround. 

* **Describe the solution you'd like**

Combine different charts to make an interactive bubble chart.

* **Describe alternatives you've considered**

Having a different type of chart, but I don't like it. 

* **Additional context**

### PR DESCRIPTION
**Pull request by TASM-Ilianna-Pirounaki**
## Proposed changes

Just some random changes to text files as part of the training session exercise. 

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [X] New feature (change which adds functionality)
- [ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [X] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [X] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?

@EHENDERS1

### CODE REVIEW

File: new_file [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Hello again :)
```
Comment (DIPAD-Fran-Bryden) [inline new_file:1]: I like the use of :)

File: new_file [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+ 
+This is a test file 
+
+Bye
```
Comment (DIPAD-Fran-Bryden) [inline new_file:5]: Unnecessary comment

**Review by DIPAD-Fran-Bryden — APPROVED** [review]
This is the best code I have ever seen

File: text_file.txt [hunk 3]
```diff
@@ -3,3 +3,6 @@ This is a random text file. You can add and remove things to it to test how git
 
 This is some text.
 This is some other text.
+
+
+Hello :)
```
Comment (EHENDERS1) [inline text_file.txt]: Hi :)

**Review by EHENDERS1 — APPROVED** [review]
Nice text!

**Pull request closed without merging**